app/pages/summary/complete/ui-tests/general.ui.test.js

```diff
@@ -82,6 +82,16 @@ test('should render the description', async (t) => {
     .expect(await extractInnerText(description)).eql(content.description);
 });
 
+test('should not render the orderDescription for print page', async (t) => {
```

Comment:
can you change this to the page description. It does render the orderDescription